quantecon/tests/test_robustlq.py

```diff
@@ -46,9 +46,9 @@ def setUp(self):
 
         # the *_pf endings refer to an example with pure forecasting
         # (see p171 in Robustness)
-        self.rblq_test = RBLQ(Q, R, A, B, C, beta, theta)
-        self.rblq_test_pf = RBLQ(Q_pf, R, A, B_pf, C, beta, theta)
-        self.lq_test = LQ(Q, R, A, B, C, beta)
+        self.rblq_test = RBLQ(Q, R, A, B, C, beta=beta, theta=theta)
+        self.rblq_test_pf = RBLQ(Q_pf, R, A, B_pf, C, beta=beta, theta=theta)
```

Comment:
Are the changes in these two lines really necessary?

Reply:
I see they are not. I will fix this.